WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -56,25 +46,25 @@ - (void)viewDidLoad
     CGRect frame = self.view.frame;
     frame = CGRectMake(0.0f, 0.0f, frame.size.width, frame.size.height);
     self.scrollView = [[UIScrollView alloc] initWithFrame:frame];
-    _scrollView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
-    _scrollView.maximumZoomScale = 4.0f;
-    _scrollView.minimumZoomScale = 0.1f;
-    _scrollView.scrollsToTop = NO;
-    _scrollView.delegate = self;
-    [self.view addSubview:_scrollView];
+    self.scrollView.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
+    self.scrollView.maximumZoomScale = 4.0f;
+    self.scrollView.minimumZoomScale = 0.1f;
```

Comment:
Would it make sense to make these constants as a part of the clean up. We could probably drop the `f` and let the compiler handle the type.